PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,214 @@
+using module PSKoans
+[Koan(Position = 105)]
+param()
+<#
+    Cmdlet Verbs
+
+    The fundamental building block of PowerShell are cmdlets and functions.
+
+    We'll cover functions later, but cmdlets are made up of the following syntax:
+
+        Verb-Noun
+
+    With the verb being an action, and the noun representing the target of the command.
+
+    In this Koan, we'll cover the 5 basic verbs which'll see you through the majority of your
+    PowerShell needs. But PowerShell has more than just 5 predefined verbs! You can see them
+    all by running the following cmdlet:
+
+        Get-Verb
+#>
+
+Describe "Basic Verbs" {
+
+    Context "Get" {
+        <#
+            Get
+
+            Cmdlets with the Get verb are used for retrieving data.
+
+            So for example "Get-Process" will retrieve all the processes running on 
+            your current machine.
+        #>
+
+        It 'is for commands that retrieve data'  {
+            <#
+                Using the Get-Command cmdlet, which retrieves all available commands,
+                find 5 new Get-* commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Get cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Get).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.count -eq 5 | 
+            Should -BeTrue -Because "five unique cmdlets should be supplied"
```

Comment:
This probably doesn't need to be on its own line, I think? The same with the others. 🙂

Reply:
Changed in AboutCmdletVerbs and AboutGetMember